presto-main/src/main/java/com/facebook/presto/memory/NodeMemoryConfig.java

```diff
@@ -24,8 +24,10 @@
 public class NodeMemoryConfig
 {
     public static final String QUERY_MAX_MEMORY_PER_NODE_CONFIG = "query.max-memory-per-node";
+    public static final String QUERY_MAX_REVOCABLE_MEMORY_PER_NODE_CONFIG = "query.max-revocable-memory-per-node";
```

Comment:
Make value of this property to follow value of `query.max-memory-per-node` as long only `query.max-memory-per-node` is specified in config.